server/mailet/mock-smtp-server/src/main/java/org/apache/james/mock/smtp/server/model/Condition.java

```diff
@@ -75,6 +75,27 @@ public final int hashCode() {
         }
     }
 
+    class MatchNothingCondition implements Condition {
```

Comment:
Modify the commit title s/operator/condition/

Reply:
I wrote this class for using it in the tests but I realize it doesn't mean anything. A behavior is supposed to be matched not ignored. I will remove this class anyway